src/megaapi_impl.cpp

```diff
@@ -14875,6 +14902,21 @@ void MegaApiImpl::getua_result(TLVstore *tlv, attr_t type)
                 }
                 break;
             }
+            case MegaApi::USER_ATTR_DEVICE_NAMES:
+            {
+                const char *buf = stringMap->get(client->getDeviceid().c_str());
+                if (!buf)
+                {
+                    e = API_ENOENT;
+                    break;
+                }
+                else
+                {
+                    request->setName(Base64::atob(buf).c_str());
+                }
+
```

Comment:
The `if` above has a break, so no need for an `else` block here :)
```suggestion cpp
                request->setName(Base64::atob(buf).c_str());
```